### PR TITLE
magento/merchdocs#1124: Describe sign out of the customer account

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes sign out of the customer account

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/customers/customer-sign-in.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=sig

## Additional information

Customer Sign In | Magento 2 4 User Guide 2021-04-01 17-47-52

### CODE REVIEW

File: src/customers/customer-sign-in.md [hunk 1]
```diff
@@ -81,3 +81,14 @@ _Sign In_
 
    ![]({% link images/images-b2b/customer-company-dashboard.png %}){: .zoom}
    _Account Dashboard - company_
+
+## Sign out of Customer Account
+
+1. In the upper-right corner, click the **Welcome, Customer Name!** <span class="btn">∨</span> icon.
```
Comment (atwixfirster) [inline src/customers/customer-sign-in.md:87]: @ZhovnirMaria could you please use `top right` instead of `upper-right`. 

Thank you!

**Comment by dobooth** [comment]
Is this image necessary?

**Comment by atwixfirster** [comment]
Hi, @dobooth !

> Is this image necessary?

Visualization helps readers to understand a content better. However, in this particular case we can skip the image. 

Up to you, @dobooth ! ;)

Thank you!

File: src/customers/customer-sign-in.md [hunk 2]
```diff
@@ -81,3 +81,14 @@ _Sign In_
...
+
+1. Click the **Sign Out** button.
```
Comment (leslietilling) [inline src/customers/customer-sign-in.md:89]: We don't say "the X button, and instead use button formatting

```suggestion
1. Click <span class="btn">Sign Out</span>.
```

File: src/customers/customer-sign-in.md (hunk 1 above)
Comment (leslietilling) [inline src/customers/customer-sign-in.md:87]: To match our style...

```suggestion
1. In the upper-right corner next to  _Welcome, Customer Name!_, click  the **v** menu selector.
```

**Review by leslietilling — APPROVED** [review]

File: src/customers/customer-sign-in.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/customers/customer-sign-in.md]: @atwixfirster  
Done. Thank you!

**Comment by atwixfirster** [comment]
@leslietilling  , could you please add the next labels to this PR:

- `partners-contribution`
- `Partner: Atwix`

Thank you!

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1125`.

**Pull request merged**